plugins/system_upgrade.py

```diff
@@ -71,7 +71,10 @@
 
 
 def reboot():
-    Popen(["systemctl", "reboot"])
+    if os.getenv("DNF_SYSTEM_UPGRADE_NO_REBOOT", False):
```

Comment:
I had to look up what the ``False`` is there for. How about to pass it as a keyword argument?

Reply:
It's pretty common for getter's second argument to be the default, but I've added it, makes it a bit clearer.

Reply:
Thanks. Otherwise, it looks good.